### PR TITLE
Add delete favoris end

### PR DESCRIPTION
**Pull request by ilalande**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by ThomasDCW — APPROVED** [review]
ok

File: backend/src/models/FavoriteManager.js [hunk 1]
```diff
@@ -3,25 +3,25 @@ const AbstractManager = require("./AbstractManager");
...
+      [userId]
     );
   }
 
-  delete(numberLessonId, userLessonId, id) {
+  delete(lessonId, userId) {
     return this.connection.query(
-      `delete userFavLesson from userFavLesson INNER JOIN user ON userFavLesson.user_id=user.id WHERE user_id = ? AND lesson_id = ? AND user.id = ?`,
-      [userLessonId, numberLessonId, id]
+      `DELETE FROM userFavLesson $ WHERE lesson_id = ? AND user_id = ?`,
```
Comment (LuigiMenez) [inline backend/src/models/FavoriteManager.js:23]: Nop $

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

**Review by ThomasDCW — APPROVED** [review]
ok

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LoicBrassart — COMMENTED** [review]
SnE

**Pull request merged**